contracts/plugins/assets/RTokenAsset.sol

```diff
@@ -72,9 +135,8 @@ contract RTokenAsset is IAsset {
     /// @return {UoA/tok} The lower end of the price estimate
     /// @return {UoA/tok} The upper end of the price estimate
     function price() public view virtual returns (uint192, uint192) {
-        try this.tryPrice() returns (uint192 low, uint192 high) {
-            assert(low <= high);
-            return (low, high);
+        try this.prices() returns (Price memory p, Price memory) {
```

Comment:
seems like price() and lotPrice() are now way more inefficient?  necessary unfortunate side-effect or can we still have tryPrice() for some efficiency?

Reply:
They're definitely more inefficient. My thought process was that `RTokenAsset.price()` only has a single use left: `RevenueTrader.manageToken()`. We don't call it from BackingManager anymore, and since RTokenAsset doesn't have price checks in `refresh()` we don't call it during `assetRegistry.refresh()`. 

But I think this logic fails in two ways:
1. Collateral that use the RTokenAsset price, such as `CvxStableRTokenMetapoolCollateral`,  _does_ call the price during refresh(). 
2. It also bundles the revert behavior of `price()`/`lotPrice()` together, which we don't want. Very possible for the underlying collateral's `price()` to revert and for `lotPrice()` not to. We should maintain that property here.

All to say, adding `tryPrice()` back...